test/unit-test/src/cfpb-forms/src/organisms/Multiselect-spec.js

```diff
@@ -79,7 +79,7 @@ describe( 'Multiselect', () => {
       search.click();
       simulateEvent( 'keydown', search, { keyCode: 40 } );
 
-      expect( document.activeElement.id ).toBe( 'Debt collection' );
+      expect( document.activeElement.value ).toBe( 'Debt collection' );
```

Comment:
Maybe we should also check the id. I think this should be correct based on the review changes:

`expect( document.activeElement.id ).toBe( 'test-select-debt-collection' );`